src/metabase/api/search.clj

```diff
@@ -558,12 +559,26 @@
                              :offset              mw.offset-paging/*offset*
                              :search-native-query search_native_query
                              :verified            verified}))
-        duration   (- (System/currentTimeMillis) start-time)]
-    ;; Only track global searches
-    (when (and (nil? models)
-               (nil? table_db_id)
-               (not archived))
-      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id* {:runtime-milliseconds duration}))
+        duration   (- (System/currentTimeMillis) start-time)
+        has-advanced-filters (some some?
+                                   [models created_by created_at last_edited_by
+                                    last_edited_at search_native_query verified])]
+    (when (some? context)
+      (when (and (nil? table_db_id) (not archived))
```

Comment:
I actually think you can remove the check on this line, since what we really want to do is track searches from either the search bar or app.

It's sufficient to use 
```
(when (contains? #{"filtered-search" "global-search"} context) ...) 
``` 
as the condition for sending a track-event

Reply:
Ah you're right

Reply:
Also, why can't we just call these what they are? i.e. "search-bar" or "search-app". I keep forgetting what they are

Reply:
Wait filtered-search should always be filtered

but yeah I actually had them as "search-bar" and "search-app"